### PR TITLE
Add new FAH release.

### PR DESCRIPTION
**Pull request by sanjabonic**

### CODE REVIEW

File: source/blog/2018-02-28-fedora-atomic-28-feb-18.html.md [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+author: dustymabe
+date: 2018-02-28 00:00:00 UTC
+published: true
+comments: false
+tags: atomic, fedora
+---
+
+Fedora Atomic Host 27.93 is available. We have a new kernel (4.15), [ostree](https://github.com/ostreedev/ostree/releases/tag/v2018.2), and [rpm-ostree](https://github.com/projectatomic/rpm-ostree/releases/tag/v2018.3) in this release.
+
+It is also worth noting that now the `rpm-ostree status` output will prefix the remote:ref with `ostree://` in order to denote the system is following an ostree repository remote (see example below). This is in preparation of some upstream changes related to rpm-ostree jigdo, where updates can be delivered via a special rpm in a yum repo rather than an ostree server/remote.
```
Comment (jlebon) [inline source/blog/2018-02-28-fedora-atomic-28-feb-18.html.md]: So, we're actually in the process of renaming this to `rojig` :). Can we do `s/jigdo/rojig/`? Also maybe a link to the high-level issue: https://github.com/projectatomic/rpm-ostree/issues/1081 ?

Comment (dustymabe) [reply source/blog/2018-02-28-fedora-atomic-28-feb-18.html.md]: what about `rpm-ostree jigdo (AKA rojig)` OR `rojig (previously known as rpm-ostree jigdo)` ?

Comment (sanjabonic) [reply source/blog/2018-02-28-fedora-atomic-28-feb-18.html.md]: Edit it once it's decided and let's get this out - I'll merge when I see a LGTM from @dustymabe.

**Comment by dustymabe** [comment]
@jlebon actually there are quite a few features in the last two releases of rpm-ostree that would be great to have a more in-depth blog post on (automatic updates, etc). Should we do it as part of this announcement or in a separate blog post?

**Comment by sanjabonic** [comment]
I vote for separate blog post because more fuel for interaction.

File: source/blog/2018-02-28-fedora-atomic-28-feb-18.html.md (hunk 1 above)
Comment (jlebon) [reply source/blog/2018-02-28-fedora-atomic-28-feb-18.html.md]: A `git grep` shows we haven't mentioned jigdo at all on the site before right? Let's not even mention it once to reduce search engine noise?

**Comment by jlebon** [comment]
Yeah, we're overdue for an rpm-ostree blog post. :) Will try to get one out soon.

**Comment by dustymabe** [comment]
didn't mean to delete the branch, fixed.. i'll merge now

**Pull request merged**